docs/maps/clean-data.asciidoc

```diff
@@ -0,0 +1,183 @@
+[role="xpack"]
+[[maps-clean-your-data]]
+=== Clean your data
+
+// https://github.com/elastic/kibana/issues/135319
+
+Geospatial fields in {es} have certain restrictions that need to be addressed before upload. On this section a few recipes will be presented to help troubleshooting common issues on this type of data.
+
+[float]
+==== Convert to GeoJSON or Shapefile
+
+With https://gdal.org/programs/ogr2ogr.html[ogr2ogr] (part of the https://gdal.org[GDAL/OGR] suite) it is pretty straight forward to convert datasets from dozens of formats into a GeoJSON or Esri Shapefile. For example, converting a GPX file can be achieved with the following commands:
```

Comment:
How about "Use https://gdal.org/programs/ogr2ogr.html[ogr2ogr] (part of the https://gdal.org[GDAL/OGR] suite) to convert datasets into aGeoJSON or Esri Shapefile. For example, use the following commands to convert a GPX file into GeoJSON"